app/views/govuk_publishing_components/components/docs/accordion.yml

```diff
@@ -181,21 +181,49 @@ examples:
                     <a class="govuk-link" href="#">Retiring your service</a>
                   </li>
               </ul>'
+  with_ga4_tracking:
+    description: |
+      Allows you to add GA4 tracking to an accordion. This will add a data module and data-attributes with JSONs to the accordion. See the `ga4-event-tracker` docs for more information.
+    data:
+      ga4_tracking: true
+      items:
+        - heading:
+            text: Writing well for the web
+          content:
+            html: <p class="govuk-body">This is the content for Writing well for the web.</p>
+          data_attributes:
+            custom_data_attr: custom-data-attr-accordion-item-1
+        - heading:
+            text: Writing well for specialists
+          content:
+            html: <p class="govuk-body">This is the content for Writing well for specialists.</p>
+          data_attributes:
+            custom_data_attr: custom-data-attr-accordion-item-2
+        - heading:
+            text: Know your audience
+          content:
+            html: <p class="govuk-body">This is the content for Know your audience.</p>
+          data_attributes:
+            custom_data_attr: custom-data-attr-accordion-item-3
+        - heading:
```

Comment:
Nitpick: not sure you need this much data to demonstrate this example, a couple of steps would be fine.

Reply:
Thanks @andysellick - good point. I've made it 2 steps.